src/lib/terminal.js

```diff
@@ -68,7 +70,7 @@ class Terminal {
 
   setCursorPosition(line, col){
     const { lines } = this.state;
-    for(var ix = lines.length; ix <= line; ix++){
+    for(let ix = lines.length; ix <= line; ix++){
```

Comment:
Fill won't work in this case, fill doesn't work past the bounds of the existing array and this loop is for filling in past the end of the array.